### PR TITLE
Translated all lessons to tamil language and added ta.json as a tamil translation for quizapp 

### PR DESCRIPTION
**Pull request by ndrohith09**
Translated all lessons to tamil language

### CODE REVIEW

**Comment by jlooper** [comment]
hi, thank you - the localized code for Tamil seems to be 'ta' so please rename the references to that in the files, thank you!

**Review by jlooper — CHANGES_REQUESTED** [review]
we don't need to change that as the file is named `.github`

**Review by jlooper — CHANGES_REQUESTED** [review]
let's change all references to Tamil to be 'ta'

File: 3-terrarium/translations/README.tam.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# என் நிலப்பரப்பு:  ஜாவாஸ்கிரிப்டைப் பயன்படுத்தி ஹெச்டிஎம்எல், சிஎஸ்எஸ் மற்றும் டோம் கையாளுதல் பற்றி அறிய ஒரு திட்டம் 🌵🌱
+
+ஒரு சிறிய இழுவை மற்றும் குறியீடு தியானம் கைவிட.ஒரு சிறிய ஹெச்டிஎம்எல், ஜேஎஸ் மற்றும் சிஎஸ்எஸ் மூலம், நீங்கள் ஒரு வலை இடைமுகத்தை உருவாக்கலாம், அதை பாணிசெய்யலாம், மற்றும் ஒரு தொடர்பு சேர்க்கலாம்.
+
+![என் நிலப்பரப்பு](../images/screenshot_gray.png)
+
+# பாடங்கள்
+
+1. [ஹெச்டிஎம்எல் அறிமுகம்](../1-intro-to-html/translations/README.tam.md)
```
[the remark this answers is not in the document]
Comment (ndrohith09) [reply 3-terrarium/translations/README.tam.md]: Yes , I will .Thank you!

**Comment by ndrohith09** [comment]
All changes which were requested were performed.Thank You!

**Review by jlooper — CHANGES_REQUESTED** [review]
Please roll back the edit to .github

**Comment by ndrohith09** [comment]
Status

- [X] 1
- [X] 2
- [X] 2
- [X] 3
- [X] 4
- [X] 5
- [X] 6
- [X] 7
- [X] Quiz

**Comment by ndrohith09** [comment]
> Please roll back the edit to .github

Are you asking about the checklist for translation?

**Comment by jlooper** [comment]
> > Please roll back the edit to .github
> 
> Are you asking about the checklist for translation?

no, it's the edit you made to change the spelling of the folder name from `.github` to `github` in the original file

**Comment by ndrohith09** [comment]
> > > Please roll back the edit to .github
> > 
> > 
> > Are you asking about the checklist for translation?
> 
> no, it's the edit you made to change the spelling of the folder name from `.github` to `github` in the original file

Yes ! Sorry . I modified it. You can review now

**Review by jlooper — APPROVED** [review]

**Pull request merged**